railties/test/generators/scaffold_generator_test.rb

```diff
@@ -96,6 +97,18 @@ def test_scaffold_on_invoke
     assert_file "app/assets/stylesheets/product_lines.css"
   end
 
+  def test_scaffold_on_invoke
```

Comment:
What is the intent of the test? It is the duplicated name with L12 and the duplicated assertions with L68-L74.

Reply:
🤦‍♂️ My mistake, I forgot to remove this.